test/Dialect/Calyx/errors.mlir

```diff
@@ -1050,3 +1050,99 @@ module attributes {calyx.entrypoint = "main"} {
     }
   }
 }
+
+// ----- 
+
+module attributes {calyx.entrypoint = "main"} {
+  calyx.component @main(%go: i1 {go}, %clk: i1 {clk}, %reset: i1 {reset}) -> (%done: i1 {done}) {
+    %r.in, %r.write_en, %r.clk, %r.reset, %r.out, %r.done = calyx.register @r : i32, i1, i1, i1, i32, i1
+    calyx.wires {
+
+    }
+    calyx.control {
+      // expected-error @+1 {{'calyx.invoke' op the input for 'calyx.invoke' is empty.}}
```

Comment:
Can we make this error message more specific and say exactly how many inputs were expected for invocation? In general, an `invoke` is not required to provide assignments for all ports of the component so this is a bit confusing.

Reply:
Ah, you're right! The `out = out` threw me off.